### ISSUE TITLE
Questions/Problems about the weights

### ISSUE THREAD

**Issue by JonathanFly** [Dengsgithub/DRD-Net#1]
Hi, I was curious about this so I tried it. I may be misunderstanding how to use this properly, so I apologize if I'm just making a silly mistake here.

1. 200H works, but the effect isn't as strong as what I see in the paper (With the Putin Umbrella test, it's pretty good on the Obama one)
2. 800 returns yellow images? I think it worked one time actually, so I wonder if the input image has to be a different type?
3. 200L just gives an error, you can see which in the ipynb file I linked.

This is how I tested it, Google Colab, with my test images:

https://gist.github.com/JonathanFly/680f5ab85360c2f406b92ffcd1c345ed#file-derainingdrd-net-ipynb

**Comment by Dengsgithub** [comment]
We apologize that there are some bug in our first version code, now we update a new version code and  the usage of this code. Thanks.

**Comment by JonathanFly** [comment]
> We apologize that there are some bug in our first version code, now we update a new version code and the usage of this code. Thanks.

Thanks you for fixing it. I think I'm going to come back later and check this out, and see if I can get the same extreme results you got in the some of the paper examples. I'll close this issue.

If you are curious I tried the model that did work in video, and I know it's not designed to derain video but I tried it anyway 

https://twitter.com/jonathanfly/status/1167878374758313985
